llvm/test/LTO/AMDGPU/gpu-rdc-amdgpu-attrs.ll

```diff
@@ -2,6 +2,7 @@
 ; RUN: opt -passes="lto<O3>" -debug-only=amdgpu-attributor -S -o - %s 2>&1 | FileCheck %s --check-prefix=POST-LINK
 
 ; REQUIRES: amdgpu-registered-target
+; REQUIRES: asserts
```

Comment:
Why does it need assertion?

Reply:
I couldn't really test the debug-only option due to hardware limitations. @chapuni pointed out that assertion is required to use debug-only?

Reply:
Shilei's right, assertion is not required in the test. I will revert this patch.